packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -145,49 +150,55 @@ You need to add a DNS TXT record to your domain to verify ownership of your doma
+2. Paste the CNAME **Host** URL into the appropriate field at the registrar, for example, the **Name** or **Host** field.
 
     Often a registrar creates an A record automatically when you create a subdomain. Make sure that the CNAME record and the A record don't have the same name.
 
     > **Note:** Depending on your registrar, you may only need to enter the subdomain part. For example, if you picked the subdomain `id.example.com`, your registrar may only require you to create a CNAME record for `id` (because `.example.com` is implied). If you're not sure, check your registrar's documentation.
 
-3. Paste the value from the **Data** column that you copied in the **Create a DNS TXT record** section into the appropriate field at the registrar, for example, the **Record** or **Value** field.
+3. Paste the CNAME **Value** into the appropriate field at the registrar, for example, the **Record** or **Value** field.
 4. Save the record.
 
-After the CNAME record is saved and confirmed by your registrar, you are done setting up a custom domain name for your Okta organization. Okta begins to serve traffic over your custom domain. It may take a few minutes to propagate the changes.
+Uploading your TLS certificate is the next step in the configuration wizard.
+
+### Provide a TLS certificate
+
+Okta serves traffic over HTTPS (TLS) on your custom domain. Use this section to enter your TLS certificate, private key, and, if applicable, a certificate chain.
+
+1. On the **Certificate** page of the configuration wizard, paste your PEM-encoded public certificate for your subdomain in the **Certificate** field. Be sure to include the `----BEGIN CERTIFICATE----` and the `----END CERTIFICATE----` lines.
+2. Paste your PEM-encoded private key for your subdomain in the **Private key** field. Be sure to include the `-----BEGIN RSA PRIVATE KEY-----` and `-----END RSA PRIVATE KEY-----` lines.
+3. Enter a PEM-encoded certificate chain (if you have one) in the **Certificate chain** field. Certificate chain files can contain keys that are up to 4096 bits. The order in which the root and intermediary certificates appear in the file matters. The intermediate CA certificate should be at the top and then the root CA certificate at the bottom.
+
+    > **Note:** Android devices require a certificate chain. You must provide a certificate chain if you want your custom domain to work with apps on Android. See [Security with HTTPS and SSL](https://developer.android.com/training/articles/security-ssl) for more details.
+
+4. Click **Finish**.
 
-### Confirm that your custom URL works
+> **Note:** After you click **Finish**, it may take several minutes before your custom domain is ready.
+
+After you provide the TLS certificate, private key, and certificate chain (if applicable), you have set up a custom domain name for your Okta org. Okta begins to serve traffic over your custom domain.
+
+## Confirm that your custom URL works
```

Comment:
We should note that it can take ~15 minutes for the domain and certificate to be ready and testable.